lib/ansible/modules/windows/win_regedit.ps1

```diff
@@ -21,313 +22,317 @@
 
 $ErrorActionPreference = "Stop"
 
-$params = Parse-Args $args -supports_check_mode $true
+$params = Parse-Args -arguments $args -supports_check_mode $true
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
-$diff_support = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -default $false
+$diff_mode = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -default $false
```

Comment:
That's fair enough, makes sense when you are looking for modules that currently support it. Will add it to the agenda for the next meeting to dicuss.